x-pack/platform/plugins/shared/onechat/server/services/agents/modes/chat/prompts.ts

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { BaseMessageLike } from '@langchain/core/messages';
+import { customInstructionsBlock, formatDate } from '../utils/prompt_helpers';
+
+export const getActPrompt = ({
+  customInstructions,
+  messages,
+}: {
+  customInstructions?: string;
+  messages: BaseMessageLike[];
+}): BaseMessageLike[] => {
+  return [
+    [
+      'system',
+      `You are an helpful chat assistant from the Elasticsearch company.
```

Comment:
nit: we are "Elastic" or the company that created Elasticsearch. There is no elasticsearch company